chicken-noodle-soup.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Noodles and More Nooddles!</title>
+    <style>
+      .image {
+        width: 250px;
+        height: auto;
+      }
+    </style>
```

Comment:
great use of a `style` tag here :) however, you could consider using the `width` and `height` attribute directly in your `img` tag

https://www.w3schools.com/tags/att_img_width.asp